lazyscribe/artifacts/__init__.py

```diff
@@ -23,11 +24,30 @@ def _get_handler(alias: str) -> Type[Artifact]:
         The artifact handler class object. This object will need to be constructed
         using :py:meth:`lazyscribe.artifacts.Artifact.construct`.
     """
-    for obj in Artifact.__subclasses__():
-        if obj.alias == alias:
-            out = obj
+    eps = entry_points()
+    entry = eps.select(group="artifact_type")
```

Comment:
Hm, how do we feel about this entrypoint naming? IMO, something `lazyscribe` specific might be less likely to have a conflict with another package. Maybe ``"lazyscribe.artifact_types"``?